### PR TITLE
[4.0] com_newsfeeds sidebar

### PR DESCRIPTION
**Pull request by brianteeman**
Removes the unused sidebar code from com_newsfeeds

### CODE REVIEW

File: administrator/components/com_newsfeeds/tmpl/newsfeeds/default.php [hunk 1]
```diff
@@ -35,12 +35,7 @@
 ?>
 <form action="<?php echo Route::_('index.php?option=com_newsfeeds&view=newsfeeds'); ?>" method="post" name="adminForm" id="adminForm">
 	<div class="row">
-		<?php if (!empty($this->sidebar)) : ?>
-            <div id="j-sidebar-container" class="col-md-2">
-				<?php echo $this->sidebar; ?>
-            </div>
-		<?php endif; ?>
-        <div class="<?php if (!empty($this->sidebar)) {echo 'col-md-10'; } else { echo 'col-md-12'; } ?>">
+        <div class="col-md-12">
```
Comment (Quy) [inline administrator/components/com_newsfeeds/tmpl/newsfeeds/default.php]:
```suggestion
		<div class="col-md-12">
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b0f04ed0c7584777b462003bad42af9cfea92e17<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24867">issues.joomla.org/tracker/joomla-cms/24867</a>.</sub>

**Comment by travisrisner** [comment]
I have tested this item :white_check_mark: successfully on b0f04ed

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24867">issues.joomla.org/tracker/joomla-cms/24867</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**